src/components/pages/Home.tsx

```diff
@@ -80,20 +81,31 @@ function SectionQueryResult({
     const source = axios.CancelToken.source()
 
     async function init() {
-      try {
-        setLoading(true)
-        const result = await queryMetadata(query, source.token)
-        if (queryData && result.totalResults > 0) {
-          const searchDIDs = queryData.split(' ')
-          const sortedAssets = sortElements(result.results, searchDIDs)
-          const overflow = sortedAssets.length - 9
-          sortedAssets.splice(sortedAssets.length - overflow, overflow)
-          result.results = sortedAssets
+      if (chainIds.length === 0) {
```

Comment:
Nice, this solves something I was about to write an issue for. This use case that all chains are unselected was never thought about and simply showing empty results is way better than before